src/main/java/org/ow2/proactive/catalog/dto/CatalogObjectMetadata.java

```diff
@@ -109,7 +112,16 @@ public CatalogObjectMetadata(String bucketName, String name, String kind, String
         } else {
             this.metadataList = metadataList;
         }
+        this.projectName = getProjectNameIfExists();
+
+    }
 
+    public String getProjectNameIfExists() {
+        Optional<Metadata> projectNameIfExists = metadataList.stream()
+                                                             .filter(property -> property.getKey()
+                                                                                         .equals("project_name"))
+                                                             .findAny();
+        return projectNameIfExists.map(meta -> meta.getValue()).orElse("");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)

Reply:
I agree